src/scripts/webpack/util.js

```diff
@@ -1,12 +1,13 @@
 import path from 'path';
 
-export function getHtmlPluginConfig(NODE_ENV, title) {
+export function getHtmlPluginConfig(NODE_ENV, config) {
   return {
     inject: false,
     bundle: NODE_ENV === 'production',
     minify: NODE_ENV === 'production' ? {} : false,
     template: path.resolve(process.cwd(), './src/app/index.html'),
-    title
+    title: config.title,
```

Comment:
`title` is a [configuration](https://github.com/jantimon/html-webpack-plugin#configuration) key